Runtime/BacktraceDatabase.cs

```diff
@@ -622,7 +754,21 @@ protected virtual void LoadReports()
             {
                 return;
             }
+
             var files = BacktraceDatabaseFileContext.GetRecords();
+            if (!files.Any())
```

Comment:
`GetRecords` returns `IEnumerable`. You enumerate the collection twice, here, and in `foreach`. Maybe materialize the collection by calling `.ToArray()`?